assets/src/components/AssignmentTable.js

```diff
@@ -120,6 +120,16 @@ function AssignmentTable (props) {
     }
   }, [currentWeekRow.current])
 
+  const getDescription = (assignment) => {
```

Comment:
I think it's worth using template literals here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals. You'll be able to avoid concat-ing using template literals

Reply:
Ok I'll take another shot, I wasn't able to figure out how to reference the variables for some reason, thanks.

Reply:
```js
`
   ${assignment.goalGrade}
`
```
```js
`
   ${assignment.grade ? assignment.grade : 'Not graded'}
`
```

Both of these should work!

Reply:
Thanks, I had forgotten the $ 🤦